New resource: `azurerm_media_streaming_locator` (#9992)
terraform-providers/terraform-provider-azurerm: website/docs/r/media_streaming_locator.html.markdown

---
subcategory: "Media"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_media_streaming_locator"
description: |-

>   Manages a Streaming Locator.

Reviewer: ```suggestion
  Manages a Media Streaming Locator.
```

---

# azurerm_media_streaming_locator

> Manages a Streaming Locator.

Reviewer: ```suggestion
Manages a Media Streaming Locator.
```

  resource_group_name         = azurerm_resource_group.example.name
  media_services_account_name = azurerm_media_services_account.example.name
  description                 = "Asset description"
}

resource "azurerm_media_streaming_locator" "example" {
  name                        = "example"
  resource_group_name         = azurerm_resource_group.example.name
  media_services_account_name = azurerm_media_services_account.example.name
  asset_name                  = azurerm_media_asset.example.name
  streaming_policy_name       = "Predefined_ClearStreamingOnly"
}
```

## Arguments Reference

The following arguments are supported:

* `asset_name` - (Required) Asset Name. Changing this forces a new Streaming Locator to be created.

* `media_services_account_name` - (Required) The Media Services account name. Changing this forces a new Streaming Locator to be created.

* `name` - (Required) The name which should be used for this Streaming Locator. Changing this forces a new Streaming Locator to be created.

* `resource_group_name` - (Required) The name of the Resource Group where the Streaming Locator should exist. Changing this forces a new Streaming Locator to be created.

> * `streaming_policy_name` - (Required) Name of the Streaming Policy used by this Streaming Locator. Either specify the name of Streaming Policy you created or use one of the predefined Streaming Policies. The predefined Streaming Policies available are: 'Predefined_DownloadOnly', 'Predefined_ClearStreamingOnly', 'Predefined_DownloadAndClearStreaming', 'Predefined_ClearKey', 'Predefined_MultiDrmCencStreaming' and 'Predefined_MultiDrmStreaming'. Changing this forces a new Streaming Locator to be created.

Reviewer: ```suggestion
* `streaming_policy_name` - (Required) Name of the Streaming Policy used by this Streaming Locator. Either specify the name of Streaming Policy you created or use one of the predefined Streaming Policies. The predefined Streaming Policies available are: `Predefined_DownloadOnly`, `Predefined_ClearStreamingOnly`, `Predefined_DownloadAndClearStreaming`, `Predefined_ClearKey`, `Predefined_MultiDrmCencStreaming` and `Predefined_MultiDrmStreaming`. Changing this forces a new Streaming Locator to be created.
```
